apps/api/src/app/environments-v1/usecases/output-renderers/in-app-output-renderer.usecase.ts

```diff
@@ -9,6 +9,15 @@ export class InAppOutputRendererUsecase {
   execute(renderCommand: RenderCommand): InAppRenderOutput {
     const { skip, disableOutputSanitization, ...outputControls } = renderCommand.controlValues ?? {};
 
-    return outputControls as any;
+    if (disableOutputSanitization) {
+      return outputControls as any;
+    } else {
+      const { body, ...rest } = outputControls as any;
+
+      return {
+        ...rest,
+        body: sanitizeMessageContent(body),
```

Comment:
we want to sanitize only the possible "html" control value

Reply:
I think subject and the other fields should follow the same rules. 

An XSS can be executed with a `javascript:` vector even in the redirectURL field.

Reply:
done